exla/mix.exs

```diff
@@ -47,6 +47,7 @@ defmodule EXLA.MixProject do
   defp deps do
     [
       {:nx, path: "../nx"},
+      {:complex, "~> 0.3.0", github: "elixir-nx/complex", branch: "main"},
```

Comment:
```suggestion
```

Reply:
I we remove this, it will re-lock upon mix deps.get, it seems.